cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,29 @@
 /// <reference types='cypress' />
 
-describe('', () => {
-  before(() => {
+import HomePage from '../support/pages/HP.pageObject';
+import LaptopsPage from '../support/pages/laptops.pageObject';
+import CartPage from '../support/pages/cart.pageObject';
+import OrderPage from '../support/pages/makeOrder.pageObject';
 
-  });
+describe('Purchase Laptop', () => {
+  const homePage = new HomePage();
+  const laptopsPage = new LaptopsPage();
+  const cartPage = new CartPage();
+  const orderPage = new OrderPage();
+
+  it('Should make the purchase', () => {
+    homePage.visit();
+    homePage.clickLaptops();
+    laptopsPage.clickSony();
+    laptopsPage.addToCart();
+    laptopsPage.getAlert();
 
-  it('', () => {
+    cy.contains('Cart').click();
+    cartPage.assertCart('Sony vaio i7');
+    cartPage.clickOrder();
 
+    orderPage.fillAllFields();
```

Comment:
```suggestion
    orderPage.fillAllFields();
```
what is all fields? separate this function into different methods